packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -23,6 +28,63 @@ def get_exhibitor_admin_password():
     return password
 
 
+def _get_bundle_list(dcos_api_session):
+    response = check_json(dcos_api_session.health.get('/report/diagnostics/list/all'))
+    bundles = []
+    for _, bundle_list in response.items():
+        if bundle_list is not None and isinstance(bundle_list, list) and len(bundle_list) > 0:
+            # append bundles and get just the filename.
+            bundles += map(lambda s: os.path.basename(s['file_name']), bundle_list)
+    return bundles
+
+
+def check_json(response):
+    response.raise_for_status()
+    try:
+        json_response = response.json()
+        logging.debug('Response: {}'.format(json_response))
+    except ValueError:
+        logging.exception('Could not deserialize response contents:{}'.format(response.content.decode()))
+        raise
+    assert len(json_response) > 0, 'Empty JSON returned from dcos-diagnostics request'
```

Comment:
This function is presented as a util, but this error messages assumes the response is from dcos-diagnostics.